installer/install.sh

```diff
@@ -30,11 +59,11 @@ fi
 
 cd $(dirname $0)
 if [ -r ./machine.conf ]; then
-    . ./machine.conf
+    read_conf_file "./machine.conf"
 fi
 
-if [ -r ./onie-image-arm64.conf ]; then
-    . ./onie-image-arm64.conf
+if [ -r ./onie-image*.conf ]; then
+. ./onie-image*.conf
```

Comment:
If there are multiple files matching the wildcard, only one is executed.

```
$ . a.sh
a
$ . b.sh
b
$ . *.sh
a
```

Reply:
Seems there is a case of `onie-image.conf` and `onie-image-*.conf` both exist for one platform.

Reply:
Still applicable.